### DISCUSSION TITLE
Which is more performant: Chart or Echart

### DISCUSSION THREAD

**Discussion by JeremieLitzler** [vuepress-theme-hope/vuepress-theme-hope#2606] [Q&A] [answered]
Hello,

I was wondering why we have 2 plugins and which is more performant overall?

Thanks.

**Comment by Mister-Hope** [comment, marked as answer]
Those tools are not written by me, so I can not make a accurate answer. You may go to its own repro to see if there is any comparation.

 I do think that Echarts is more powerful and also means it's heavier.

Both tools are all run at client side, so I think when you mean website performance, bundle size is the factor which most matters.

**Reply by JeremieLitzler** [reply]
Yes, I mean bundle size and maybe also build time. So `chart` is ligher. I take note and I'll request more info to the third parties.
